requirements.in

```diff
@@ -11,3 +11,5 @@ vcsinfo>=2.1.105
 graphlib-backport>=1.0.3
 timeout-decorator>=0.5.0
 python-on-whales>=0.61.0
+# python-on-whales requires pydantic 1.10.11 08/2023
+pydantic>=1.10.11
```

Comment:
Why include this requirement if that's what python-on-whales already requires? Just curious

Reply:
I added this so that if python-on-whales is removed or replaced in the future, then there wouldn't be guesswork to make sure to add pydantic explicitly so that the config validation would work.